dali/operators/util/randomizer_impl_gpu.cuh

```diff
@@ -38,7 +39,8 @@ Randomizer<GPUBackend>::Randomizer(int seed, size_t len) {
   len_ = len;
   cudaGetDevice(&device_);
   states_ = GPUBackend::New(sizeof(curandState) * len, true);
-  initializeStates<<<128, 256>>>(len_, seed, reinterpret_cast<curandState*>(states_));
+  initializeStates<<<std::ceil(len_/static_cast<float>(block_size_)), block_size_>>>
```

Comment:
```suggestion
  initializeStates<<<div_ceil(len_, block_size_), block_size_>>>
```
using floating point arithmetic may lead to incorrect results when len_ > 256*2^23

Reply:
done